app/src/ingestion/elements.py

```diff
@@ -0,0 +1,34 @@
+from dataclasses import dataclass
+from enum import Enum
+from typing import List
+
+
+class TextType(Enum):
+    NARRATIVE_TEXT = "NarrativeText"
+    LIST_ITEM = "ListItem"
+    LIST = "List"
+
+
+@dataclass
+class Page:
+    # The actual page in the document
+    pdf_page_number: int
+
+    # The label of the page in the document
+    # E.g., "i", "ii", "1", "2", etc.
+    document_page_number: str
```

Comment:
Wonder if we'd be OK just using an `int` for the index of the page in the document rather than this class. Also I made some slight renames here for clarity but don't feel strongly about this if folks have alternative style preferences

Reply:
did we want to store the total amount of pages somewhere?

Reply:
> did we want to store the total amount of pages somewhere?

That would go in a PdfInfo or DocInfo class, if we needed it.